### PR TITLE
read me

### LINKED ISSUE #3: Read me

### PR DESCRIPTION
**Pull request by maserati12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,55 +2,56 @@ barebones setup for JavaScript / Node projects replace this with a one-liner phr
 
 [![Codacy Badge](https://img.shields.io/badge/Code%20Quality-D-red)](https://img.shields.io/badge/Code%20Quality-D-red)
 
-
 ## About
 
-What is this project about. Ok to enrich here or the section above it with an image. 
+What is this project about. Ok to enrich here or the section above it with an image.
```
Comment (maserati12) [inline README.md:7]: This is awesome

File: README.md [hunk 2]
```diff
@@ -2,55 +2,56 @@ barebones setup for JavaScript / Node projects replace this with a one-liner phr
...
 
 Once this repo has been setup on Codacy by the TTL, replace the above badge with the actual one from the Codacy dashboard, and add the code coverage badge as well. This is mandatory
 
 This is a simple JavaScript starter repo template for setting up your project. The setup contains
 
-- Jest: For runnung tests. We strongly recommend that JavaScript projects use Jest
+-   Jest: For runnung tests. We strongly recommend that JavaScript projects use Jest
```
Comment (dully-d) [inline README.md:13]: review the error maserati

**Review by maserati12 — COMMENTED** [review]
Done

**Review by dully-d — COMMENTED** [review]
done and dusted

**Review by dully-d — APPROVED** [review]
dusted

**Pull request merged**